javascript-timers/interval.js

```diff
@@ -0,0 +1,19 @@
+var $heading = document.querySelector('h1');
+
+function countdown() {
+  window.setInterval(changeText, 2000);
+}
+
+function changeText() {
+  if ($heading.textContent === '4') {
+    $heading.textContent = '3';
+  } else if ($heading.textContent === '3') {
+    $heading.textContent = '2';
+  } else if ($heading.textContent === '2') {
+    $heading.textContent = '1';
```

Comment:
This would be better with just an `if else` statement you can use `>= 1`.  You can store the text content in a variable, then decrement it, then do the if check. You want to avoid repetitive code when possible.